lib/clearance/password_strategies/sha1.rb

```diff
@@ -3,13 +3,21 @@ module PasswordStrategies
     module SHA1
       require 'digest/sha1'
 
+      DEPRECATION_MESSAGE = "[DEPRECATION] The SHA1 password strategy " \
```

Comment:
this seems like almost an exact dupe of the message in `blowfish.rb` - is there somewhere you can put this constant where the two can share it?